src/tests/test_magic_cmd.py

```diff
@@ -63,6 +65,21 @@ def ip_snippets(ip):
     yield ip
 
 
+@pytest.fixture
+def ip_with_dbapi(ip):
```

Comment:
clearer name since this also contains many types of connections, not only dbapi

```suggestion
def ip_with_connections(ip_empty):
```

also, switched the fixture to `ip_empty`, since `ip` contains sample data that you don't need for this test

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/785/files#diff-9188e75e824219dd191cdc62d6d19f2b15c4bd0cf564abe3646ea907c32cd61dR69-R80)